.github/workflows/cd-training-pipeline.yml

```diff
@@ -78,9 +78,9 @@ jobs:
     - name: 'Create TFX Training Pipeline'
       working-directory: 'training_pipeline'
       env:
-        FULL_RES_DATA: ${{ github.event.inputs.fullResolutionData }}
+        ENABLE_DATAFLOW: ${{ github.event.inputs.enableDataflow }}
       run: |
```

Comment:
Has this change been reflected in the codebase properly?

Reply:
Yup